packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts

```diff
@@ -157,18 +150,35 @@ export const buildFilterInputType = (modelType: GraphQLObjectType) => {
   });
 }
 
-export const buildCreateMutationInputType = (modelType: GraphQLObjectType) => {
+// Fetching all fields required for input type including relationships
+const getAllModelFields = (modelType: GraphQLObjectType<any, any, { [key: string]: any; }>, model: ModelDefinition) => {
```

Comment:
I think you can just have one function argument:

```suggestion
const getAllModelFields = (model: ModelDefinition) => {
```

modelType is a child of model.

Reply:
Yep. Autogenerated. I'm adding aggregate for pagination now

Reply:
Yes. This was autogenerated by IDE